src/main/java/com/techiteasy/techiteasycontrolleruitwerkingen/repositories/TelevisionRepository.java

```diff
@@ -0,0 +1,7 @@
+package com.techiteasy.techiteasycontrolleruitwerkingen.repositories;
+
+import com.techiteasy.techiteasycontrolleruitwerkingen.models.Television;
+import org.springframework.data.jpa.repository.JpaRepository;
+
+public interface TelevisionRepository extends JpaRepository<Television, Long> {
+}
```

Comment:
In principe goed, maar er is geen findBy methode door je aangemaakt, hoefde volgens mij niet voor de opdracht, maar had gekund om te oefenen. Zoals deze (komt uit de uitwerkingen) :
List<Television> findAllTelevisionsByBrandEqualsIgnoreCase(String brand);

Reply:
Hallo Ilse,

Bedankt voor je review. Ik heb de twee kleine puntjes al aangepast of verwerkt.
BIj deze snap ik niet helemaal waarom ik hier een methode zou maken genaamd findBy. En dan vanuit waar ik dit aan kan roepen en gebruiken.

Reply:
Lekker bezig! Volgens mij was het ook niet nodig (stond niet in de opdrachtomschrijving), maar stel je wilt alle TVs van een specifiek merk zoeken, dan kan dat dus met deze: List findAllTelevisionsByBrandEqualsIgnoreCase(String brand);